apps/protocol-frontend/src/components/ContributionsByDateShell.tsx

```diff
@@ -0,0 +1,37 @@
+import useContributionCountInRange from '../hooks/useContributionCount';
+import ContributionsByDateChart from './ContributionsByDateChart';
+import { TODAY_DATE, YEAR } from '../utils/constants';
+import { subWeeks, startOfDay, endOfDay } from 'date-fns';
+
+interface ContributionByDateShellProps {
+  guildIds: number[];
+}
+
+const ContributionByDateShell = ({
+  guildIds,
+}: ContributionByDateShellProps) => {
+  const {
+    isFetching,
+    isLoading,
+    isError,
+    data: contributionsByDate,
+  } = useContributionCountInRange({
+    id: null,
```

Comment:
Does null vs undefine matter here?

Reply:
Ah okay so it does then was trying to understand if that is how we have it on the backend